release-notes/opensearch-ml-common.release-notes-2.14.0.0.md

```diff
@@ -9,6 +9,7 @@ Compatible with OpenSearch 2.14.0.0
 
 ### Enhancements
 * Change httpclient to async ([#1958](https://github.com/opensearch-project/ml-commons/pull/1958))
+* Change httpclient to async bug fix ([#2418](https://github.com/opensearch-project/ml-commons/pull/2418))
```

Comment:
Let's use the commit message `Fix user defined preprocess function missing prediction issue`